### ISSUE TITLE
Feature: Opt to "Send a silent message"

### ISSUE THREAD

**Issue by erlend-sh** [RocketChat/Rocket.Chat#9017]
I'm aware of #5524 and #8806. This is something different.

Story:
Using a simple hotkey or this menu:
<img width="225" alt="screen shot 2017-12-05 at 19 39 18" src="https://user-images.githubusercontent.com/583842/33624348-6e83dbfc-d9f4-11e7-9fba-3ee103cc54b6.png">
.. I can opt to send a *silent message*. How it works is quite simple: *Silent messages* will not create notifications of any kind.

Could be useful in many contexts, but the main use case for me is private messages. So often in Slack I want to drop a team mate an asynchronous note without disrupting their current activity. To my knowledge, there's no way around this, so long as they're not currently in DnD mode. So to be on the safe side, I will send the equivalent of a DnD message.

**Comment by HammyHavoc** [comment]
Yep, this would be wonderful. I don't like to take attention away from important tasks, but do want to say things for when people have a chance to see it.

**Comment by graywolf336** [comment]
Is there any way you can see this being abused?

**Comment by graywolf336** [comment]
I didn't have any scenario in my head, I was just wondering if there was anything.

We could have it enabled only on direct message rooms by default and a setting to enable it in other room types. :thinking:

**Comment by HammyHavoc** [comment]
Sounds good to me, though do think it needs a symbol indicating that it was a silent message, that way users won't get confused and think notifications aren't functioning. :- ) Eases them into the concept of a new feature. I imagine it would look/function similarly to OTR.

**Comment by graywolf336** [comment]
What would be a good icon? :thinking:

**Comment by HammyHavoc** [comment]
Something similar to this, I think: [https://icons8.com/icon/5782/silent](https://icons8.com/icon/5782/silent)

Perhaps my colleague @bryandesrosiers would be interested in contributing a graphic to RocketChat?

**Comment by bryandesrosiers** [comment]
I'm absolutely up for helping created some kind of graphic as an icon!

**Comment by geekgonecrazy** [comment]
I love this idea. 

As long as its clearly marked as a silent message.  I can see drama being caused by someone saying: "I told you about it last night" even though used silent notification.
